README.md

```diff
@@ -6,12 +6,16 @@ Take a peek into your Rails application.
 
 ![Preview](https://f.cloud.github.com/assets/79995/244991/03cee1fa-8a74-11e2-8e33-283cf1298a60.png)
 
-This is a profiling tool originally built at GitHub to help us get an insight into our application. Now, we have extracted this into Peek, so that other Rails application can experience the same benefit.   
+No explanation of Peek is complete without discussing what Rails is. Rails is a framework written in the Ruby language for use in developing web applications. 
```

Comment:
I agree with @jacobbednarz. If this was an explanation that Peek is exclusively for Ruby on Rails (and not say Sinatra) that would be different.